### DISCUSSION TITLE
更換瀏覽器

### DISCUSSION THREAD

**Discussion by Clare0808** [1101-GS4719-C/Python-Community-GS4719-C#2025] [General]
下午打開安裝好的jupyterlab，發現畫面是一片空白，試了幾個方法後都沒用。
之後有想到會不會是瀏覽器的問題，因為原本預設的瀏覽器是IE，想說換成Chrome試試看。
但我自己其實也不知道該怎麼做，後來與電腦比較厲害的長輩討論後，他有教我照下面的步驟做更改……

1.點IE右上角的“齒輪”按鈕
2.點選“網際網路選項”
3.點選“程式“
4.點“將IE設定為預設瀏覽器”
5.選擇Chrome瀏覽器
6.點選“將此程式設定成預設值”
7.按下“確定”
完成！

以上是我解決問題的過程，有需要的人可以參考參考～

**Comment by Clare0808** [comment]
一開始上網找方法更換時，都是要在命令提示字元寫更換瀏覽器的程式，但試了好幾次都失敗。經過指導後，才知道原來有這麼簡單的方法存在。

**Comment by peresonacifer** [comment]
設置預設瀏覽器方面，因為我是Windows11，我常用的是打開「設定」>「應用程式」>「預設應用程式」，然後點選你想要預設的網頁瀏覽器，接著看到最上面有個欄位叫將.....作為預設瀏覽器，最後再點選設定預設值即可。

**Comment by CtyceT** [comment]
個人使用的是Macbook Air，瀏覽器經常會用Firefox（火狐）和Google Chrome，預設是用Chrome。但每一次要換成Firefox或者原裝的Safari時，都會跳一個建議更換預設瀏覽器的提醒。過去我都是直接忽略，直到看到這個文章才想起有這回事w

**Comment by hunteryang0108** [comment]
謝謝你提供的方法，我認為這個方法十分受用。我也提供一個更方便的方法，基本上各家瀏覽器為了增加自己的使用率，在點開使用時都會跳出詢問是否設定為預設瀏覽器的詢問視窗，只要按下確定的按鈕並跳轉幾個頁面，正在使用的瀏覽器就會變成預設瀏覽器囉。
